### ISSUE TITLE
Indentation issue with inline comments in sequences

### ISSUE THREAD

**Issue by kyle-figure** [adrienverge/yamllint#704]
Hello,

When an inline comment is in a sequence, Yaml formatters like Prettier and the Redhat YAML VSCode extension format it like so:
```yaml
---
foo: [
    1, # Comment
  ]
```
However, this produces the following warning and error from yamllint 1.35.1.

Command: `yamllint test.yaml`
Output:
```
test.yaml
  3:8       warning  too few spaces before comment  (comments)
  4:3       error    wrong indentation: expected 0 but found 2  (indentation)
```
I confirmed that test.yaml can be loaded with Python's Yaml package.

Is yamllint using an overly strict indentation check here? Or is it the formatters' fault for not aligning the closing brace with the previous block of code?

**Comment by andrewimeson** [comment]
Why use the compact/JSON notation if you're going to break it into multiple lines anyway? This is more concise and the more "YAML" way of doing the same thing.

```yaml
---
foo:
  - 1  # Comment
```

**Comment by kyle-figure** [comment]
That is a good tip. We can just change it to:
```yaml
foo:
  - [1, 2, 3, 4, 5, 6, 7] # Descriptive comment 1
  - [1, 2, 3, 4, 5, 6, 7] # Descriptive comment 2
```
Now we only get warnings about there being too few spaces before the comments.
Thanks!
